### ISSUE TITLE
[FEATURE] Create a reporting feature that gathers all the key info from HO for bug reports

### ISSUE THREAD

**Issue by tychobrailleur** [ho-dev/HattrickOrganizer#1358]
**Is your feature request related to a problem? Please describe.**
When users report bugs, it can be difficult to get information on their setup, as they may not be knowledgeable enough to collect some of the data.  This feature would allow them to simple copy-paste a report somewhere to put into GH.  This could potentially also automatically submit the bug report in the correct format.

**Describe the solution you'd like**
In the Help menu we should had a menu item which, once clicked, collects all the relevant info (OS, JVM version, timezone, logs location, database location, team ID etc. etc.), and display it in a popup for copying.  Users can then paste that report into GH bug report. 

**Additional context**
N/A

**Comment by akasolace** [comment]
I would also suggest to facilitate dumping of database. 
For exemple the GUI asking if the user wants to add DB as attachement in the ticket, opening the correct location on the system ....

**Comment by tychobrailleur** [comment]
> I would also suggest to facilitate dumping of database. For exemple the GUI asking if the user wants to add DB as attachement in the ticket, opening the correct location on the system ....

Good point.  That would also allow us to remove access token secrets before they are published publicly: they really should be removed.

**Comment by wsbrenk** [comment]
I'm also in favor of not storing the token in the database anymore - I'm afraid many people don't know what they're doing when they give us the database. In fact, it also makes sense to exchange the tokens - but then explicitly with the hint to regenerate it as soon as possible (possibly also with a command in the HO menu).

**Comment by wsbrenk** [comment]
@tychobrailleur what do you think about an github user called HOBot, who will create github issues when HO displays fatal errors and the users accepted this creation: 

https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#create-an-issue

**Comment by tychobrailleur** [comment]
> @tychobrailleur what do you think about an github user called HOBot, who will create github issues when HO displays fatal errors and the users accepted this creation:
> 
> https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#create-an-issue

I am not too fond of that idea, as it would require to store an access token in HO, which would be easy to recover.  If we wanted to automate the ticket submission, [OAuth](https://docs.github.com/en/developers/apps/building-oauth-apps) would probably be preferable (“While the API provides multiple methods for authentication, we strongly recommend using [OAuth](https://docs.github.com/en/apps/building-integrations/setting-up-and-registering-oauth-apps) for production applications.”), but this would assume that the HO user has a GH account.

I would prefer a more low-key approach where the report is generated in HO, ready to copy-paste into a GH bug report. Less hassle than having to protect that HOBot credentials.

**Comment by wsbrenk** [comment]
related to #1973

**Comment by tychobrailleur** [comment]
> related to #1973

Thanks!  I was thinking along the same line as myself, that's good. 😄
